libshaderc_spvc/src/common_shaders_for_test.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018 The Shaderc Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef COMMON_SHADERS_FOR_TESTS_H_
+#define COMMON_SHADERS_FOR_TESTS_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+const uint32_t kShader1[] = {
```

Comment:
Please add a comment to say what original source was.  That will help us understand what it's testing.

Reply:
Also, later tests probably should use SPIR-V assembly to exercise wonky cases.  Although maybe the wonky case tests really belong in SPIRV-Cross.  spvc is about plumbing.